nomad/testing.go

```diff
@@ -38,7 +38,7 @@ func TestACLServer(t testing.T, cb func(*Config)) (*Server, *structs.ACLToken) {
 func TestServer(t testing.T, cb func(*Config)) *Server {
 	// Setup the default settings
 	config := DefaultConfig()
-	config.Build = "0.7.0+unittest"
```

Comment:
Should probably just import `version` and do `config.Build = version.Version + "+unittest"` but no big deal.